README.md

```diff
@@ -26,8 +26,6 @@ const container = document.querySelector('#publish-container');
 
 const publish = new BeeyPublish(container, {
   media: {
-    url: '<url of audio/vidoe file>',
```

Comment:
Why did you remove this line? The media URL is still a valid property, isn't it?

Reply:
I must have done that by mistake. Sorry! I will fix it in the next commit.